ResearchKit/ActiveTasks/ORKStroopResult.h

```diff
@@ -66,6 +66,11 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, copy, nullable) NSString *colorSelected;
 
+/**
+ The Stroop style displayed - "box" or "text"
+ */
+@property (nonatomic, copy, nullable) NSString *stroopStyle;
```

Comment:
Is there a reason for this to be NSString instead of ORKStroopStyle?

Reply:
I was debating this - since this should only be a box or text. Have updated things to use the ORKStroopStyle enum now.

Reply:
Oh, I see how this works. It's not the style of the overall Stroop test, but the style of an individual prompt within the test. So your original design using a string may make more sense than repurposing the ORKStroopStyle enum.